services/github/github-api-provider.js

```diff
@@ -34,6 +51,7 @@ class GithubApiProvider {
     if (this.withPooling) {
       this.standardTokens = new TokenPool({ batchSize: 25 })
       this.searchTokens = new TokenPool({ batchSize: 5 })
+      this.graphqlTokens = new TokenPool({ batchSize: 25 })
```

Comment:
Is 25 a sensible number here? I don't really have a handle on the reasons for picking 5 and 25 above.

Reply:
I think that's fine. 25 was picked as a reasonable number of requests to make before re-evaluating how much quota was left, in order to share fairly across other tokens and because the way this is implemented you can't accurately evaluate how much quota is left while requests are in flight. For search I picked 5 because the limit is something tiny like 50 requests per token.